docs/ibis-for-pandas-users.ipynb

```diff
@@ -46,7 +46,7 @@
    "id": "afd136cd-18b5-48a2-8359-c47cde181c7b",
```

Comment:
<p>I don't have a suggestion for better wording here, but this isn't <em>strictly</em> correct. <code>memtable</code> is a tool for wrapping a dataframe and making it part of the expression tree so it can be used with any backend.   In this context what you're saying is true (and maybe for this tutorial the user doesn't need the full explanation), but I worry that a sentence like this will make users think <code>memtable</code> is only for working with the duckdb backend.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ibis-project/ibis/pull/6048/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>How about:</p><p> </p><blockquote>You can do this more concisely with <code>memtable</code>,which also works with other backends.  </blockquote>

Reply:
Yeah, I think that makes sense. IMO the point of this doc should be:

- Show how to load pandas dataframes into ibis
- Show how to load local files (csv, parquet) directly into ibis, mimicking normal pandas workloads
- Compare/contrast how ibis spells common query operations with their pandas equivalents